openpype/hosts/maya/plugins/publish/extract_camera_alembic.py

```diff
@@ -35,7 +39,6 @@ def process(self, instance):
 
         # validate required settings
         assert isinstance(step, float), "Step must be a float value"
-        camera = cameras[0]
 
         # Define extract output file path
         dir_path = self.staging_dir(instance)
```

Comment:
BTW we need to ensure the directory exists here.
```suggestion
        dir_path = self.staging_dir(instance)
        if not os.path.exists(dir_path):
            os.makedirs(dir_path)
```

Reply:
The `staging_dir` method creates the temporary directory - so we shouldn't actually. Right?

Reply:
Yup, definitely sounds like an issue with custom staging directory then 👍